markdown/org/docs/designs/onyx/options/armpitEase/en.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Armpit ease"
+---
+
+![The armpit ease option on Onyx](armpitease.svg)
```

Comment:
This seems to refer to a non-existing svg file?

Reply:
That's probably caused by me erroneously using a camelCase directory name. Hopefully fixing that has fixed that problem.

Reply:
What I mean is: This SVG file is not part of this PR. And since it's a new option, it's not in the repo either.
So if you have it locally, you should add it to the PR.

Reply:
This problem is actually a lot more widespread.
44 options of Onyx reference an SVG file:
```
joost@machine:~/git/freesewing/markdown/org/docs/designs/onyx/options$ grep -R svg | grep en.md | wc -l
44
```

Yet none of them actually provide said file:
```
joost@machine:~/git/freesewing/markdown/org/docs/designs/onyx/options$ ls * | grep svg | wc -l
0
```

So, what is your intent here?
- If you intent to provide SVG files to illustrate the option, please add them
- If you did not intent to do that, then don't include non-existing images in the documentation

Reply:
I thought that the file was autogenerated? I'll see if I can remember how they're generated and run the build script that does that, or just remove the .svg links if that's not the case.

Reply:
The script is 'node ./scripts/optiondocsimages.mjs', but it tries to looks in the old patterns directory, which fails since it's the designs directory now.

If this functionality is old or deprecated, I'll go ahead and remove the .svg links. In my opinion, the functionality in the v3 lab covers what these .svg files did, and better.

Reply:
> the functionality in the v3 lab covers what these .svg files did, and better

Yes, that's why these images have been removed in all docs.
The script is still there, but it's no longer in use.